src/modules/folder/folder.usecase.ts

```diff
@@ -24,6 +24,12 @@ const invalidName = /[\\/]|^\s*$/;
 
 type SortParams = Array<[SortableFolderAttributes, 'ASC' | 'DESC']>;
 
+export class NotFoundFolderException extends NotFoundException {
+  constructor() {
```

Comment:
Please use [this](https://github.com/internxt/drive-server-wip/blob/cc313904b564f55293fb6d0ce1b2482acf548ead/src/modules/user/user.usecase.ts#L47) so the `instanceof` check works properly